.travis.yml

```diff
@@ -8,8 +8,10 @@ php:
   - 5.4
 
 install:
-  - composer install --dev --prefer-source
+  - composer install --prefer-source
 
 script:
   - bin/phpunit
   - bin/psecio-parse scan src tests
+
+dist: trusty
```

Comment:
Without it, "precise" is used (for some reason, not sure why), which has stopped supporting HHVM, causing that test to fail.
If you want me to move it, I will.